src/components/Dashboard/PatientCard/index.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
 import styled from 'styled-components';
-import { useHistory } from 'react-router';
+// import { useHistory } from 'react-router';
```

Comment:
please remove comments, leave code. As this is temporary change create config prop that will hide necessary parts of the page

Reply:
we cannot leave unused Code. commit will fail (Lint)